src/sagemaker/image_uris.py

```diff
@@ -30,6 +30,7 @@
 
 ECR_URI_TEMPLATE = "{registry}.dkr.{hostname}/{repository}"
 HUGGING_FACE_FRAMEWORK = "huggingface"
+TRAINIUM_ALLOWED_FRAMEWORKS = "pytorch"
```

Comment:
We will need to grab this from json file. Can we add this action to the pending work post re-invent.